src/components/Pages/ActionsPage/ActionsPage.js

```diff
@@ -205,10 +205,24 @@ class ActionsPage extends React.Component {
       this.searchIsActiveSoFindContentThatMatch() ||
       applyTagsAndGetContent(this.props.actions, this.state.checked_values);
 
+    const steps = [
+      {
+        target: "#test-action-cards-wrapper",
+        title: "Actions chosen by your neighbors",
+        content:
+          "There are a lot of them! You can browse, or you can filter by area, impact or cost. Interested in an action? Click on any card for more info.",
+        placement: "auto",
+        spotlightClicks: true,
+        disableBeacon: true,
+        disableOverlayClose: true,
+      },
+      // ...
+    ];
+
     return (
       <>
         <ProductTour
-          steps={this.state.steps}
+          steps={steps}
```

Comment:
So I think we can take steps out of the state variables?

Reply:
yes. I think that'll work